python/huggingfaceserver/huggingfaceserver/vllm/utils.py

```diff
@@ -37,15 +36,15 @@ def vllm_available() -> bool:
 
 def infer_vllm_supported_from_model_architecture(
     model_config_path: Union[Path, str],
-) -> Optional[Type[nn.Module]]:
+) -> bool:
     if not _vllm:
-        return None
+        return False
     model_config = AutoConfig.from_pretrained(model_config_path)
     architecture = model_config.architectures[0]
```

Comment:
don't we need to check all here?

Reply:
We generally noticed a case of a single arch in response. 
Updated, with changes to check all.